starlette_admin/views.py

```diff
@@ -222,6 +222,7 @@ class BaseModelView(BaseView):
     exclude_fields_from_detail: Sequence[str] = []
     exclude_fields_from_create: Sequence[str] = []
     exclude_fields_from_edit: Sequence[str] = []
+    exclude_actions_from_detail: Sequence[str] = []
```

Comment:
We should consider adding an option to exclude from the list as well.
Just a quick thought - you could consider including the flags in the action decorator itself, but it's not a big deal if you decide not to.

Reply:
Yes, I though about adding that, but the PR was already pretty big, and I wanted to make sure the direction was good before starting that. I can add it now to make it complete.